Makefile

```diff
@@ -560,7 +560,6 @@ SOURCE_FILES = \
   SpirvIR.cpp \
   SplitTuples.cpp \
   StageStridedLoads.cpp \
-  StmtToHtml.cpp \
```

Comment:
Also need to remove the header file at https://github.com/halide/Halide/blob/c94e9e7452c0fca22205935f0b204f49cdd3dfef/Makefile#L741